mars/dataframe/merge/merge.py

```diff
@@ -245,6 +255,32 @@ def _gen_shuffle_chunks(
             )
         return reduce_chunks
 
+    @classmethod
+    def _gen_bloom_filter_chunks(cls, left, right, left_on, right_on, op):
```

Comment:
Type annotation, and seems like this function returns two tileables instead of `chunks` in the function name.